user_docs/en/userGuide.t2t

```diff
@@ -2866,6 +2874,13 @@ Specifically, following issues can be solved by running this tool:
 ++ Reload plugins ++[ReloadPlugins]
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
 
+You can also find out whether the  application that you focus has an app module integrated in NVDA or not.

```

Comment:
You can know if an application has an appModule or not, but you cannot know if this module belongs to NVDA core or to an add-on.

```suggestion
You can also find out whether the  application that you focus has an associated app module.

```